go/gildedrose/Factory.go

```diff
@@ -0,0 +1,20 @@
+package gildedrose
+
+const (
+	brie     = "Aged Brie"
+	sulfuras = "Sulfuras, Hand of Ragnaros"
+	tafkal   = "Backstage passes to a TAFKAL80ETC concert"
+)
+
+func getUpdater(item *Item) (updater Updater) {
```

Comment:
How will we add a new item to the shop? Change the factory code, recompile, and deploy a new version?

Can you find a *very cheap* - but better - solution?

Reply:
Still requires a re-compile to ship an update to customers - seems riskt

Reply:
we can make a micro service and they will communicate via rest